foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/AbstractMigrationTest.java

```diff
@@ -22,6 +25,17 @@ public abstract class AbstractMigrationTest {
 
     private static final String TEST_FILES_FOLDER = Paths.get("target", "test-classes", "test-files").toString();
 
+    protected static Boolean validateMigration(File original, File migrated) {
+        Boolean isMigrated = false;
```

Comment:
A: I'd use the primitive just so you don't make callers wonder if you're going to return `null`.